conf/views/chargeA/whatYouWillNeed.njk

```diff
@@ -0,0 +1,36 @@
+{% extends "includes/layout.njk" %}
+
+{% from "macros/title.njk"  import title %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+
+{% block pageTitle %}
+  {{ title(messages("chargeA.whatYouWillNeed.title")) }}
+{% endblock %}
+
+
+{% block mainContent %}
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-xl">
+          {{ messages("chargeA.whatYouWillNeed.heading") }}
+        </h1>
+
+        <p class="govuk-body">
+        {{ messages("chargeA.whatYouWillNeed.p1") }}
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>{{ messages("chargeA.whatYouWillNeed.li1") }}</li>
+          <li>{{ messages("chargeA.whatYouWillNeed.li2") }}</li>
+          <li>{{ messages("chargeA.whatYouWillNeed.li3") }}</li>
+        </ul>
+
+        <a href = {{nextPage}} id="submit" class="govuk-button">{{ messages("site.continue") }}</a>
+      </div>
+    </div>
+
+  <a href = "{{ viewModel.returnUrl }}" class="govuk-link"> {{ messages("return.to.link", schemeName) }}</a>
```

Comment:
`href = "{{ viewModel.returnUrl }}"` needs to be `href="{{ viewModel.returnUrl }}"`